### PR TITLE
feat: Resolve Debug to redirect to UI tool on web

### PR DESCRIPTION
**Pull request by nickybondarenko**
This PR adds a link to logger which can be used to access resolve debug tool in console.

### CODE REVIEW

File: packages/sdk/src/Confidence.ts [hunk 1]
```diff
@@ -56,6 +56,8 @@ export interface Configuration {
   /** Flag Resolver Client
    * @internal */
   readonly flagResolverClient: FlagResolverClient;
+  /* @internal */
+  readonly clientSecret: string;
```
Comment (nicklasl) [inline packages/sdk/src/Confidence.ts:60]: is it possible to declare it `private`? (not sure it helps but still)

File: packages/sdk/src/Confidence.ts [hunk 2]
```diff
@@ -314,6 +317,14 @@ export class Confidence implements EventSender, Trackable, FlagResolver {
     return (await this.evaluateFlag(path, defaultValue)).value;
   }
 
+  private showLoggerLink(flag: string, context: Context) {
+    this.config.logger.info?.(
```
Comment (nicklasl) [inline packages/sdk/src/Confidence.ts:321]: This is a too high level IMO. 
Would suggest `debug` or maybe even `trace`

File: packages/sdk/src/Confidence.ts (hunk 1 above)
Comment (nickybondarenko) [reply packages/sdk/src/Confidence.ts:60]: I don't think it's possible on an interface

File: packages/sdk/src/Confidence.ts (hunk 2 above)
Comment (nickybondarenko) [reply packages/sdk/src/Confidence.ts:321]: Chatted with Andreas about it: lower in Confidence class we only put this into info on "dev" builds. If we change it to debug/trace, the current factory instance won't show any of the logs in an app. So the user should then know to create a Confidence instance with a logger in debug mode.

Comment (nicklasl) [reply packages/sdk/src/Confidence.ts:321]: I understand, sounds good!

**Review by nicklasl — APPROVED** [review]

File: packages/sdk/src/Confidence.ts [hunk 3]
```diff
@@ -390,7 +402,7 @@ function defaultFetchImplementation(): typeof fetch {
 function defaultLogger(): Logger {
   try {
     if (process.env.NODE_ENV === 'development') {
-      return Logger.withLevel(console, 'info');
+      return Logger.withLevel(console, 'debug');
```
Comment (nicklasl) [inline packages/sdk/src/Confidence.ts:405]: Was this a conscious decision?

Comment (nickybondarenko) [reply packages/sdk/src/Confidence.ts:405]: yes! this way we default to debug logger in our demo apps and can see the logs when testing

Comment (nickybondarenko) [reply packages/sdk/src/Confidence.ts:405]: yeah that makes sense. I'll change it back to info

**Review by nicklasl — APPROVED** [review]

**Review by andreas-karlsson — APPROVED** [review]

**Pull request merged**